### DISCUSSION TITLE
'Change repository from public to private' dialog is broken in github!!!

### DISCUSSION THREAD

**Discussion by a22183nj** [community/community#48742] [Actions]
### Select Topic Area

Product Feedback

### Body

I am working on Chrome on Macbook.

When I try to change a repository of mine from public to private, there will be two pop-up dialogs. The 2nd one, which is a longer one with multiple bullet points, has a button at the bottom, where use need to click to confirm, is NOT VISIBLE!!!

I tried on my IPhone browser, same issue. Then I go to my Windows machine, I still can not see the whole button, but I can see maybe one third of it, and I can move my mouse to where I think the 'OK' is and click, only then will it work.

I can't believe this can happen on a website that's used by millions of people!! Please do something to fix it.

**Comment by sayhiben** [comment]
This is not related to Actions
